### ISSUE TITLE
Appium 1.5 beta cleanup

### ISSUE THREAD

**Issue by jlipps** [appium/appium#5539]
Once Appium 1.5 beta is out, there will be a bunch of cleanup to do to make sure everything is ready for a full release. This issue will track those tasks. Note that tasks should probably be done from the more basic modules (like appium-support) to the more inclusive modules (like appium itself).

* https://github.com/appium/appium
  * [x] Ensure all relevant existing functional tests pass with Appium 1.5
  * [x] Migrate existing functional tests to the appropriate new driver
  * [x] Go through docs and remove reference to any deprecated functionality
  * [x] Go through docs and add information about new features (the --desired-caps server flag, for example)
  * [x] Update the changelog with any breaking changes
  * [x] Go through all commits that landed in master since the 1.5 branch was created, and ensure that each one's change is reflected in the appropriate place in the 1.5 architecture
  * [x] Ensure the README is up-to-date in terms of getting started instructions
  * [x] Ensure the CONTRIBUTING doc is up-to-date, and add a section explaining the Appium package architecture (i.e., the list of all the packages that make up Appium and how they are related)
  * [x] Add a code of conduct doc to help define good ways of interacting within the community
  * [x] Make sure all deps are up to date (and there are no unused deps)
  * [x] Take a look at all TODOs to see if we need to resolve any for the release
  * [x] Set up code coverage
  * [x] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [x] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [x] Ensure that we have high unit test code coverage percentage
  * [x] Ensure that we have sufficient/appropriate functional test coverage
  * [x] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [x] Ensure that code conforms to our style
  * [x] Ensure we are not using any sync functions
  * [x] Ensure that code is well-commented
* https://github.com/appium/appium-adb
  * [x] Make sure all deps are up to date (and there are no unused deps)
  * [x] Take a look at all TODOs to see if we need to resolve any for the release
  * [x] Set up code coverage
  * [x] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [x] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [x] Ensure that we have high unit test code coverage percentage
  * [x] Ensure that we have sufficient/appropriate functional test coverage
  * [x] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [x] Ensure that code conforms to our style
  * [x] Ensure we are not using any sync functions
  * [x] Ensure that code is well-commented
  * [x] Ensure that non-production code or assets are added to `.npmignore` to reduce npm install size
* https://github.com/appium/appium-android-bootstrap
  * [x] Make sure all deps are up to date (and there are no unused deps)
  * [x] Take a look at all TODOs to see if we need to resolve any for the release
  * [x] Set up code coverage
  * [x] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [x] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [x] Ensure that we have high unit test code coverage percentage
  * [x] Ensure that we have sufficient/appropriate functional test coverage
  * [x] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [x] Ensure that code conforms to our style
  * [x] Ensure we are not using any sync functions
  * [x] Ensure that code is well-commented
* https://github.com/appium/appium-android-driver
  * [x] Make sure all deps are up to date (and there are no unused deps)
  * [x] Take a look at all TODOs to see if we need to resolve any for the release
  * [x] Set up code coverage
  * [x] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [x] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [x] Ensure that we have high unit test code coverage percentage
  * [x] Ensure that we have sufficient/appropriate functional test coverage
  * [x] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [x] Ensure that code conforms to our style
  * [x] Ensure we are not using any sync functions
  * [x] Ensure that code is well-commented
  * [x] Ensure that non-production code or assets are added to `.npmignore` to reduce npm install size
* https://github.com/appium/appium-base-driver
  * [x] Make sure all deps are up to date (and there are no unused deps)
  * [x] Take a look at all TODOs to see if we need to resolve any for the release
  * [x] Set up code coverage
  * [x] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [x] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [x] Ensure that we have high unit test code coverage percentage
  * [x] Ensure that we have sufficient/appropriate functional test coverage
  * [x] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [x] Ensure that code conforms to our style
  * [x] Ensure we are not using any sync functions
  * [x] Ensure that code is well-commented
  * [x] Ensure that non-production code or assets are added to `.npmignore` to reduce npm install size
* https://github.com/appium/appium-chromedriver
  * [x] Make sure all deps are up to date (and there are no unused deps)
  * [x] Take a look at all TODOs to see if we need to resolve any for the release
  * [x] Set up code coverage
  * [x] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [x] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [x] Ensure that we have high unit test code coverage percentage
  * [x] Ensure that we have sufficient/appropriate functional test coverage
  * [x] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [x] Ensure that code conforms to our style
  * [x] Ensure we are not using any sync functions
  * [x] Ensure that code is well-commented
  * [x] Ensure that non-production code or assets are added to `.npmignore` to reduce npm install size
* https://github.com/appium/appium-cookies
  * [x] Make sure all deps are up to date (and there are no unused deps)
  * [x] Take a look at all TODOs to see if we need to resolve any for the release
  * [x] Set up code coverage
  * [x] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [x] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [x] Ensure that we have high unit test code coverage percentage
  * [x] Ensure that we have sufficient/appropriate functional test coverage
  * [x] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [x] Ensure that code conforms to our style
  * [x] Ensure we are not using any sync functions
  * [x] Ensure that code is well-commented
  * [x] Ensure that non-production code or assets are added to `.npmignore` to reduce npm install size
* https://github.com/appium/appium-doctor
  * [x] Make sure all deps are up to date (and there are no unused deps)
  * [x] Take a look at all TODOs to see if we need to resolve any for the release
  * [x] Set up code coverage
  * [x] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [x] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [x] Ensure that we have high unit test code coverage percentage
  * [x] Ensure that we have sufficient/appropriate functional test coverage
  * [x] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [x] Ensure that code conforms to our style
  * [x] Ensure we are not using any sync functions
  * [x] Ensure that code is well-commented
  * [x] Ensure that non-production code or assets are added to `.npmignore` to reduce npm install size
* https://github.com/appium/appium-express
  * [x] Make sure all deps are up to date (and there are no unused deps)
  * [x] Take a look at all TODOs to see if we need to resolve any for the release
  * [x] Set up code coverage
  * [x] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [x] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [x] Ensure that we have high unit test code coverage percentage
  * [x] Ensure that we have sufficient/appropriate functional test coverage
  * [x] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [x] Ensure that code conforms to our style
  * [x] Ensure we are not using any sync functions
  * [x] Ensure that code is well-commented
  * [x] Ensure that non-production code or assets are added to `.npmignore` to reduce npm install size
* https://github.com/appium/appium-fake-driver
  * [ ] Make sure all deps are up to date (and there are no unused deps)
  * [ ] Take a look at all TODOs to see if we need to resolve any for the release
  * [ ] Set up code coverage
  * [ ] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [ ] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [ ] Ensure that we have high unit test code coverage percentage
  * [ ] Ensure that we have sufficient/appropriate functional test coverage
  * [ ] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [ ] Ensure that code conforms to our style
  * [ ] Ensure we are not using any sync functions
  * [ ] Ensure that code is well-commented
  * [ ] Ensure that non-production code or assets are added to `.npmignore` to reduce npm install size
* https://github.com/appium/appium-jsonwp-proxy
  * [x] Make sure all deps are up to date (and there are no unused deps)
  * [x] Take a look at all TODOs to see if we need to resolve any for the release
  * [x] Set up code coverage
  * [x] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [x] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [x] Ensure that we have high unit test code coverage percentage
  * [x] Ensure that we have sufficient/appropriate functional test coverage
  * [x] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [x] Ensure that code conforms to our style
  * [x] Ensure we are not using any sync functions
  * [x] Ensure that code is well-commented
  * [x] Ensure that non-production code or assets are added to `.npmignore` to reduce npm install size
* https://github.com/appium/logger
  * [x] Make sure all deps are up to date (and there are no unused deps)
  * [x] Take a look at all TODOs to see if we need to resolve any for the release
  * [x] Set up code coverage
  * [x] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [x] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [x] Ensure that we have high unit test code coverage percentage
  * [x] Ensure that we have sufficient/appropriate functional test coverage
  * [x] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [x] Ensure that code conforms to our style
  * [x] Ensure we are not using any sync functions
  * [x] Ensure that code is well-commented
  * [x] Ensure that non-production code or assets are added to `.npmignore` to reduce npm install size
* https://github.com/appium/appium-instruments
  * [x] Make sure all deps are up to date (and there are no unused deps)
  * [x] Take a look at all TODOs to see if we need to resolve any for the release
  * [x] Set up code coverage
  * [x] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [x] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [x] Ensure that we have high unit test code coverage percentage
  * [x] Ensure that we have sufficient/appropriate functional test coverage
  * [x] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [x] Ensure that code conforms to our style
  * [x] Ensure we are not using any sync functions
  * [x] Ensure that code is well-commented
  * [x] Ensure that non-production code or assets are added to `.npmignore` to reduce npm install size
* https://github.com/appium/appium-ios-driver
  * [x] Make sure all deps are up to date (and there are no unused deps)
  * [x] Take a look at all TODOs to see if we need to resolve any for the release
  * [x] Set up code coverage
  * [x] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [x] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [x] Ensure that we have high unit test code coverage percentage
  * [x] Ensure that we have sufficient/appropriate functional test coverage
  * [x] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [x] Ensure that code conforms to our style
  * [x] Ensure we are not using any sync functions
  * [x] Ensure that code is well-commented
  * [x] Ensure that non-production code or assets are added to `.npmignore` to reduce npm install size
* https://github.com/appium/appium-ios-simulator
  * [x] Make sure all deps are up to date (and there are no unused deps)
  * [x] Take a look at all TODOs to see if we need to resolve any for the release
  * [x] Set up code coverage
  * [x] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [x] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [x] Ensure that we have high unit test code coverage percentage
  * [x] Ensure that we have sufficient/appropriate functional test coverage
  * [x] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [x] Ensure that code conforms to our style
  * [x] Ensure we are not using any sync functions
  * [x] Ensure that code is well-commented
  * [x] Ensure that non-production code or assets are added to `.npmignore` to reduce npm install size
* https://github.com/appium/appium-remote-debugger
  * [x] Make sure all deps are up to date (and there are no unused deps)
  * [x] Take a look at all TODOs to see if we need to resolve any for the release
  * [x] Set up code coverage
  * [x] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [x] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [x] Ensure that we have high unit test code coverage percentage
  * [x] Ensure that we have sufficient/appropriate functional test coverage
  * [x] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [x] Ensure that code conforms to our style
  * [x] Ensure we are not using any sync functions
  * [x] Ensure that code is well-commented
  * [x] Ensure that non-production code or assets are added to `.npmignore` to reduce npm install size
* https://github.com/appium/appium-selendroid-driver
  * [x] Make sure all deps are up to date (and there are no unused deps)
  * [x] Take a look at all TODOs to see if we need to resolve any for the release
  * [x] Set up code coverage
  * [x] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [x] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [x] Ensure that we have high unit test code coverage percentage
  * [x] Ensure that we have sufficient/appropriate functional test coverage
  * [x] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [x] Ensure that code conforms to our style
  * [x] Ensure we are not using any sync functions
  * [x] Ensure that code is well-commented
  * [x] Ensure that non-production code or assets are added to `.npmignore` to reduce npm install size
* https://github.com/appium/appium-support
  * [x] Make sure all deps are up to date (and there are no unused deps)
  * [x] Take a look at all TODOs to see if we need to resolve any for the release
  * [x] Set up code coverage
  * [x] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [x] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [x] Ensure that we have high unit test code coverage percentage
  * [x] Ensure that we have sufficient/appropriate functional test coverage
  * [x] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [x] Ensure that code conforms to our style
  * [x] Ensure we are not using any sync functions
  * [x] Ensure that code is well-commented
  * [x] Ensure that non-production code or assets are added to `.npmignore` to reduce npm install size
* https://github.com/appium/appium-uiauto
  * [x] Make sure all deps are up to date (and there are no unused deps)
  * [x] Take a look at all TODOs to see if we need to resolve any for the release
  * [x] Set up code coverage
  * [x] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [x] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [x] Ensure that we have high unit test code coverage percentage
  * [x] Ensure that we have sufficient/appropriate functional test coverage
  * [x] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [x] Ensure that code conforms to our style
  * [x] Ensure we are not using any sync functions
  * [x] Ensure that code is well-commented
  * [x] Ensure that non-production code or assets are added to `.npmignore` to reduce npm install size
* https://github.com/appium/appium-uiautomator
  * [x] Make sure all deps are up to date (and there are no unused deps)
  * [x] Take a look at all TODOs to see if we need to resolve any for the release
  * [x] Set up code coverage
  * [x] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [x] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [x] Ensure that we have high unit test code coverage percentage
  * [x] Ensure that we have sufficient/appropriate functional test coverage
  * [x] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [x] Ensure that code conforms to our style
  * [x] Ensure we are not using any sync functions
  * [x] Ensure that code is well-commented
  * [x] Ensure that non-production code or assets are added to `.npmignore` to reduce npm install size
* https://github.com/appium/appium-xcode
  * [x] Make sure all deps are up to date (and there are no unused deps)
  * [x] Take a look at all TODOs to see if we need to resolve any for the release
  * [x] Set up code coverage
  * [x] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [x] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [x] Ensure that we have high unit test code coverage percentage
  * [x] Ensure that we have sufficient/appropriate functional test coverage
  * [x] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [x] Ensure that code conforms to our style
  * [x] Ensure we are not using any sync functions
  * [x] Ensure that code is well-commented
  * [x] Ensure that non-production code or assets are added to `.npmignore` to reduce npm install size
* https://github.com/appium/authorize-ios
  * [x] Make sure all deps are up to date (and there are no unused deps)
  * [x] Take a look at all TODOs to see if we need to resolve any for the release
  * [x] Set up code coverage
  * [x] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [x] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [x] Ensure that we have high unit test code coverage percentage
  * [x] Ensure that we have sufficient/appropriate functional test coverage
  * [x] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [x] Ensure that code conforms to our style
  * [x] Ensure we are not using any sync functions
  * [x] Ensure that code is well-commented
  * [x] Ensure that non-production code or assets are added to `.npmignore` to reduce npm install size
* https://github.com/appium/node-mobile-json-wire-protocol
  * [x] Make sure all deps are up to date (and there are no unused deps)
  * [x] Take a look at all TODOs to see if we need to resolve any for the release
  * [x] Set up code coverage
  * [x] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [x] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [x] Ensure that we have high unit test code coverage percentage
  * [x] Ensure that we have sufficient/appropriate functional test coverage
  * [x] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [x] Ensure that code conforms to our style
  * [x] Ensure we are not using any sync functions
  * [x] Ensure that code is well-commented
  * [x] Ensure that non-production code or assets are added to `.npmignore` to reduce npm install size
* https://github.com/appium/node-simctl
  * [x] Make sure all deps are up to date (and there are no unused deps)
  * [x] Take a look at all TODOs to see if we need to resolve any for the release
  * [x] Set up code coverage
  * [x] Set up CI (unit tests, functional tests, code coverage; should be on all relevant platforms, and should include multi-executor parallelism for long-running testsuites)
  * [x] Ensure that we are using `appium-support` functions, not home-grown helpers
  * [x] Ensure that we have high unit test code coverage percentage
  * [x] Ensure that we have sufficient/appropriate functional test coverage
  * [x] Write a decent README, explaining (a) what the purpose of the module is, (b) how it relates to other appium modules, and (c) how to use its interface
  * [x] Ensure that code conforms to our style
  * [x] Ensure we are not using any sync functions
  * [x] Ensure that code is well-commented
  * [x] Ensure that non-production code or assets are added to `.npmignore` to reduce npm install size
* Once all above updates have been made
  * [x] bump the version of each above package to the next major version (e.g., 1.0 or 2.0), so we can begin using semver. Do this in the appropriate order so we can cascade the versions up
* Publishing strategy
  * [ ] rename 'master' branch to 'old-master'
  * [ ] rename '1.5' branch to 'master'
  * [ ] test and publish appium 1.5, new docs etc...
  * [ ] create new '1.5' branch which will track patches for 1.5

**Comment by timqian** [comment]
When will Appium 1.5 be released?

**Comment by scottdixon** [comment]
Working on Android Driver cleanup.

**Comment by jlipps** [comment]
@timqian it will be released when the tasks in this issue are complete and it has been adequately tested. there is no date set for that.

**Comment by Floodzie2** [comment]
Do you have a likely release data for a beta of 1.5? We'd like to test performance vs 1.4.

**Comment by jlipps** [comment]
1.5 beta is out now. `npm install -g appium@beta`

**Comment by Jonahss** [comment]
@jonahss is doing tasks for appium-support

**Comment by imurchie** [comment]
I will take on `appium-cookies`.

**Comment by Floodzie2** [comment]
@jlipps - great, thank you very much!!

**Comment by Floodzie2** [comment]
I am unable to install the beta, here is the output - is anyone else having similar issues?

npm WARN engine xmlbuilder@2.2.1: wanted: {"node":"0.8.x || 0.10.x"} (current: {"node":"0.12.0","npm":"2.5.1"})
npm WARN deprecated MD5@1.3.0: deprecated, use lowercase 'md5@2.x' from now on
npm WARN engine xmlbuilder@2.2.1: wanted: {"node":"0.8.x || 0.10.x"} (current: {"node":"0.12.0","npm":"2.5.1"})
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-base-driver requires adm-zip@'^0.4.7' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/adm-zip,
npm WARN unmet dependency which is version 0.4.4
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-base-driver requires babel-runtime@'=5.5.5' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/babel-runtime,
npm WARN unmet dependency which is version 5.8.20
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-base-driver requires request@'^2.61.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/request,
npm WARN unmet dependency which is version 2.45.0
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-base-driver requires uuid-js@'^0.7.5' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/uuid-js,
npm WARN unmet dependency which is version 0.7.4
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-express requires babel-runtime@'5.5.5' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/babel-runtime,
npm WARN unmet dependency which is version 5.8.20
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-express requires body-parser@'^1.12.4' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/body-parser,
npm WARN unmet dependency which is version 1.9.0
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-express requires colors@'^1.1.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/colors,
npm WARN unmet dependency which is version 1.0.3
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-express requires express@'^4.12.4' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/express,
npm WARN unmet dependency which is version 4.9.7
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-express requires method-override@'^2.3.3' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/method-override,
npm WARN unmet dependency which is version 2.2.0
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-express requires morgan@'^1.5.3' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/morgan,
npm WARN unmet dependency which is version 1.3.2
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-express requires serve-favicon@'^2.2.1' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/serve-favicon,
npm WARN unmet dependency which is version 2.1.5
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-express requires source-map-support@'^0.2.10' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/source-map-support,
npm WARN unmet dependency which is version 0.3.2
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-adb requires appium-support@'1.1.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-support,
npm WARN unmet dependency which is version 2.0.0-beta21
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-adb requires ncp@'~2.0.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/ncp,
npm WARN unmet dependency which is version 1.0.1
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-adb requires rimraf@'~2.3.4' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/rimraf,
npm WARN unmet dependency which is version 2.2.8
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-android-bootstrap requires appium-adb@'2.0.0-beta13' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-adb,
npm WARN unmet dependency which is version 2.0.0-beta14
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-android-bootstrap requires mobile-json-wire-protocol@'^0.12.3' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/mobile-json-wire-protocol,
npm WARN unmet dependency which is version 0.22.6
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-chromedriver requires appium-support@'^1.1.1' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-support,
npm WARN unmet dependency which is version 2.0.0-beta21
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-chromedriver requires q@'^1.2.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/q,
npm WARN unmet dependency which is version 1.0.1
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-chromedriver requires request@'^2.57.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/request,
npm WARN unmet dependency which is version 2.45.0
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-chromedriver requires rimraf@'^2.3.3' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/rimraf,
npm WARN unmet dependency which is version 2.2.8
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-chromedriver requires through@'^2.3.7' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/through,
npm WARN unmet dependency which is version 2.3.6
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-express requires body-parser@'^1.12.4' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/body-parser,
npm WARN unmet dependency which is version 1.9.0
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-express requires colors@'^1.1.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/colors,
npm WARN unmet dependency which is version 1.0.3
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-express requires express@'^4.12.4' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/express,
npm WARN unmet dependency which is version 4.9.7
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-express requires method-override@'^2.3.3' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/method-override,
npm WARN unmet dependency which is version 2.2.0
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-express requires morgan@'^1.5.3' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/morgan,
npm WARN unmet dependency which is version 1.3.2
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-express requires serve-favicon@'^2.2.1' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/serve-favicon,
npm WARN unmet dependency which is version 2.1.5
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-express requires source-map-support@'^0.2.10' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/source-map-support,
npm WARN unmet dependency which is version 0.3.2
WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-support requires babel-runtime@'=5.6.16' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/babel-runtime,
npm WARN unmet dependency which is version 5.5.5
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-support requires bplist-creator@'0.0.6' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/bplist-creator,
npm WARN unmet dependency which is version 0.0.4
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-support requires bplist-parser@'^0.1.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/bplist-parser,
npm WARN unmet dependency which is version 0.0.5
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-support requires ncp@'^2.0.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/ncp,
npm WARN unmet dependency which is version 1.0.1
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-support requires rimraf@'^2.4.1' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/rimraf,
npm WARN unmet dependency which is version 2.2.8
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/asyncbox/node_modules/es6-mapify/node_modules/traceur requires glob@'4.3' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/glob,
npm WARN unmet dependency which is version 4.0.6
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/asyncbox/node_modules/es6-mapify/node_modules/traceur requires source-map-support@'~0.2.8' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/source-map-support,
npm WARN unmet dependency which is version 0.3.2
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-fake-driver/node_modules/appium-express requires body-parser@'^1.12.4' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/body-parser,
npm WARN unmet dependency which is version 1.9.0
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-fake-driver/node_modules/appium-express requires colors@'^1.1.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/colors,
npm WARN unmet dependency which is version 1.0.3
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-fake-driver/node_modules/appium-express requires express@'^4.12.4' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/express,
npm WARN unmet dependency which is version 4.9.7
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-fake-driver/node_modules/appium-express requires method-override@'^2.3.3' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/method-override,
npm WARN unmet dependency which is version 2.2.0
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-fake-driver/node_modules/appium-express requires morgan@'^1.5.3' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/morgan,
npm WARN unmet dependency which is version 1.3.2
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-fake-driver/node_modules/appium-express requires serve-favicon@'^2.2.1' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/serve-favicon,
npm WARN unmet dependency which is version 2.1.5
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-ios-driver/node_modules/appium-ios-simulator requires bplist-creator@'^0.0.6' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/bplist-creator,
npm WARN unmet dependency which is version 0.0.4
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-ios-driver/node_modules/appium-ios-simulator requires bplist-parser@'^0.1.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/bplist-parser,
npm WARN unmet dependency which is version 0.0.5
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-ios-driver/node_modules/appium-ios-simulator requires glob@'^5.0.14' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/glob,
npm WARN unmet dependency which is version 4.0.6
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-ios-driver/node_modules/appium-support requires babel-runtime@'=5.5.5' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/babel-runtime,
npm WARN unmet dependency which is version 5.8.20
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-ios-driver/node_modules/appium-support requires bplist-creator@'0.0.6' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/bplist-creator,
npm WARN unmet dependency which is version 0.0.4
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-ios-driver/node_modules/appium-support requires bplist-parser@'^0.1.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/bplist-parser,
npm WARN unmet dependency which is version 0.0.5
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-ios-driver/node_modules/appium-support requires ncp@'^2.0.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/ncp,
npm WARN unmet dependency which is version 1.0.1
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-ios-driver/node_modules/appium-support requires rimraf@'^2.4.1' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/rimraf,
npm WARN unmet dependency which is version 2.2.8
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/md5calculator/node_modules/unzip/node_modules/pullstream requires readable-stream@'~1.0.31' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/md5calculator/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/md5calculator/node_modules/unzip/node_modules/match-stream requires readable-stream@'~1.0.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/md5calculator/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/md5calculator/node_modules/unzip/node_modules/pullstream/node_modules/slice-stream requires readable-stream@'~1.0.31' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/md5calculator/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/unzip/node_modules/pullstream requires readable-stream@'~1.0.31' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/unzip/node_modules/match-stream requires readable-stream@'~1.0.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/unzip/node_modules/pullstream/node_modules/slice-stream requires readable-stream@'~1.0.31' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/lazystream requires readable-stream@'~1.0.2' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/tar-stream requires readable-stream@'^1.0.27-1' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/zip-stream requires readable-stream@'~1.0.26' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/tar-stream/node_modules/bl requires readable-stream@'~1.0.26' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/zip-stream/node_modules/compress-commons requires readable-stream@'~1.0.26' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/zip-stream/node_modules/compress-commons/node_modules/crc32-stream requires readable-stream@'~1.0.24' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/yiewd/node_modules/wd/node_modules/archiver/node_modules/tar-stream requires readable-stream@'^1.0.27-1' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/yiewd/node_modules/wd/node_modules/archiver/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/yiewd/node_modules/wd/node_modules/archiver/node_modules/zip-stream requires readable-stream@'~1.0.26' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/yiewd/node_modules/wd/node_modules/archiver/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/yiewd/node_modules/wd/node_modules/archiver/node_modules/lazystream requires readable-stream@'~1.0.2' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/yiewd/node_modules/wd/node_modules/archiver/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/yiewd/node_modules/wd/node_modules/archiver/node_modules/tar-stream/node_modules/bl requires readable-stream@'~1.0.26' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/yiewd/node_modules/wd/node_modules/archiver/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/yiewd/node_modules/wd/node_modules/archiver/node_modules/zip-stream/node_modules/crc32-stream requires readable-stream@'~1.0.24' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/yiewd/node_modules/wd/node_modules/archiver/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/unzip/node_modules/pullstream requires readable-stream@'~1.0.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.26-4
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/unzip/node_modules/match-stream requires readable-stream@'~1.0.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.26-4
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/unzip/node_modules/pullstream/node_modules/slice-stream requires readable-stream@'~1.0.2' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.26-4
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/md5calculator/node_modules/unzip/node_modules/pullstream requires readable-stream@'~1.0.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/md5calculator/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.26-4
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/md5calculator/node_modules/unzip/node_modules/match-stream requires readable-stream@'~1.0.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/md5calculator/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.26-4
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/md5calculator/node_modules/unzip/node_modules/pullstream/node_modules/slice-stream requires readable-stream@'~1.0.2' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/md5calculator/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.26-4
npm WARN unmet dependency /usr/local/lib/node_modules/md5calculator/node_modules/unzip/node_modules/pullstream requires readable-stream@'~1.0.31' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/md5calculator/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/md5calculator/node_modules/unzip/node_modules/match-stream requires readable-stream@'~1.0.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/md5calculator/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/md5calculator/node_modules/unzip/node_modules/pullstream/node_modules/slice-stream requires readable-stream@'~1.0.31' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/md5calculator/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
Error: Cannot find module 'MD5'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (lib/fs.js:3:31)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (lib/util.js:3:20)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
npm WARN deprecated MD5@1.3.0: deprecated, use lowercase 'md5@2.x' from now on
npm WARN engine xmlbuilder@2.2.1: wanted: {"node":"0.8.x || 0.10.x"} (current: {"node":"0.12.0","npm":"2.5.1"})
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-base-driver requires adm-zip@'^0.4.7' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/adm-zip,
npm WARN unmet dependency which is version 0.4.4
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-base-driver requires babel-runtime@'=5.5.5' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/babel-runtime,
npm WARN unmet dependency which is version 5.8.20
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-base-driver requires request@'^2.61.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/request,
npm WARN unmet dependency which is version 2.45.0
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-base-driver requires uuid-js@'^0.7.5' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/uuid-js,
npm WARN unmet dependency which is version 0.7.4
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-android-bootstrap requires appium-adb@'2.0.0-beta13' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-adb,
npm WARN unmet dependency which is version 2.0.0-beta14
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-android-bootstrap requires mobile-json-wire-protocol@'^0.12.3' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/mobile-json-wire-protocol,
npm WARN unmet dependency which is version 0.22.6
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-chromedriver requires appium-support@'^1.1.1' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-support,
npm WARN unmet dependency which is version 2.0.0-beta21
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-chromedriver requires q@'^1.2.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/q,
npm WARN unmet dependency which is version 1.0.1
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-chromedriver requires request@'^2.57.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/request,
npm WARN unmet dependency which is version 2.45.0
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-chromedriver requires rimraf@'^2.3.3' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/rimraf,
npm WARN unmet dependency which is version 2.2.8
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-chromedriver requires through@'^2.3.7' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/through,
npm WARN unmet dependency which is version 2.3.6
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-express/node_modules/body-parser requires bytes@'2.1.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/bytes,
npm WARN unmet dependency which is version 1.0.0
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-express/node_modules/body-parser requires bytes@'2.1.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/bytes,
npm WARN unmet dependency which is version 1.0.0
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-fake-driver/node_modules/appium-express/node_modules/body-parser requires bytes@'2.1.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/bytes,
npm WARN unmet dependency which is version 1.0.0
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-fake-driver/node_modules/babel-core/node_modules/regexpu/node_modules/recast requires ast-types@'0.8.11' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium/node_modules/appium-fake-driver/node_modules/babel-core/node_modules/ast-types,
npm WARN unmet dependency which is version 0.7.8
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/md5calculator/node_modules/unzip/node_modules/pullstream requires readable-stream@'~1.0.31' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/md5calculator/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/md5calculator/node_modules/unzip/node_modules/match-stream requires readable-stream@'~1.0.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/md5calculator/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/md5calculator/node_modules/unzip/node_modules/pullstream/node_modules/slice-stream requires readable-stream@'~1.0.31' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/md5calculator/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/unzip/node_modules/pullstream requires readable-stream@'~1.0.31' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/unzip/node_modules/match-stream requires readable-stream@'~1.0.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/unzip/node_modules/pullstream/node_modules/slice-stream requires readable-stream@'~1.0.31' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/lazystream requires readable-stream@'~1.0.2' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/tar-stream requires readable-stream@'^1.0.27-1' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/zip-stream requires readable-stream@'~1.0.26' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/tar-stream/node_modules/bl requires readable-stream@'~1.0.26' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/zip-stream/node_modules/compress-commons requires readable-stream@'~1.0.26' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/zip-stream/node_modules/compress-commons/node_modules/crc32-stream requires readable-stream@'~1.0.24' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/wd/node_modules/archiver/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/yiewd/node_modules/wd/node_modules/archiver/node_modules/tar-stream requires readable-stream@'^1.0.27-1' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/yiewd/node_modules/wd/node_modules/archiver/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/yiewd/node_modules/wd/node_modules/archiver/node_modules/zip-stream requires readable-stream@'~1.0.26' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/yiewd/node_modules/wd/node_modules/archiver/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/yiewd/node_modules/wd/node_modules/archiver/node_modules/lazystream requires readable-stream@'~1.0.2' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/yiewd/node_modules/wd/node_modules/archiver/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/yiewd/node_modules/wd/node_modules/archiver/node_modules/tar-stream/node_modules/bl requires readable-stream@'~1.0.26' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/yiewd/node_modules/wd/node_modules/archiver/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/yiewd/node_modules/wd/node_modules/archiver/node_modules/zip-stream/node_modules/crc32-stream requires readable-stream@'~1.0.24' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium-1.3.0-beta1/node_modules/yiewd/node_modules/wd/node_modules/archiver/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/unzip/node_modules/pullstream requires readable-stream@'~1.0.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.26-4
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/unzip/node_modules/match-stream requires readable-stream@'~1.0.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.26-4
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/unzip/node_modules/pullstream/node_modules/slice-stream requires readable-stream@'~1.0.2' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.26-4
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/md5calculator/node_modules/unzip/node_modules/pullstream requires readable-stream@'~1.0.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/md5calculator/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.26-4
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/md5calculator/node_modules/unzip/node_modules/match-stream requires readable-stream@'~1.0.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/md5calculator/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.26-4
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/md5calculator/node_modules/unzip/node_modules/pullstream/node_modules/slice-stream requires readable-stream@'~1.0.2' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/appium.0.17/node_modules/md5calculator/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.26-4
npm WARN unmet dependency /usr/local/lib/node_modules/md5calculator/node_modules/unzip/node_modules/pullstream requires readable-stream@'~1.0.31' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/md5calculator/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/md5calculator/node_modules/unzip/node_modules/match-stream requires readable-stream@'~1.0.0' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/md5calculator/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN unmet dependency /usr/local/lib/node_modules/md5calculator/node_modules/unzip/node_modules/pullstream/node_modules/slice-stream requires readable-stream@'~1.0.31' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/md5calculator/node_modules/unzip/node_modules/readable-stream,
npm WARN unmet dependency which is version 1.0.33-1
npm WARN engine xmlbuilder@2.2.1: wanted: {"node":"0.8.x || 0.10.x"} (current: {"node":"0.12.0","npm":"2.5.1"})
info iOSTestApp cleaning apps
dbug iOSTestApp cleaning app for iphonesimulator9.0
dbug iOSTestApp cleaning app for iphoneos9.0
info iOSTestApp finished cleaning apps
info iOSTestApp building apps
dbug iOSTestApp building app for iphonesimulator9.0
dbug iOSTestApp building app for iphoneos9.0
ERR! iOSTestApp Error: Command 'xcodebuild -sdk iphoneos9.0' exited with code 65
ERR! iOSTestApp     at ChildProcess.<anonymous> (lib/teen_process.js:56:19)
ERR! iOSTestApp     at ChildProcess.emit (events.js:110:17)
ERR! iOSTestApp     at maybeClose (child_process.js:1008:16)
ERR! iOSTestApp     at Process.ChildProcess._handle.onexit (child_process.js:1080:5)
ERR! iOSTestApp  { [Error: Command 'xcodebuild -sdk iphoneos9.0' exited with code 65]
ERR! iOSTestApp   stdout: 'Build settings from command line:\n    SDKROOT = iphoneos9.0\n\n=== BUILD TARGET TestApp OF PROJECT TestApp WITH THE DEFAULT CONFIGURATION (Release) ===\n\nCheck dependencies\nCode Sign error: No matching provisioning profiles found: No provisioning profiles with a valid signing identity (i.e. certificate and private key pair) matching the bundle identifier “io.appium.TestApp” were found.\n\n',
ERR! iOSTestApp   stderr: '** BUILD FAILED **\n\n\nThe following build commands failed:\n\tCheck dependencies\n(1 failure)\n',
ERR! iOSTestApp   code: 65 }
info Chromedriver Install Opening temp file to write chromedriver_mac32 to...
info Chromedriver Install Downloading http://chromedriver.storage.googleapis.com/2.18/chromedriver_mac32.zip...
info Chromedriver Install Writing binary content to /var/folders/n7/90d6rzz56fdbn_4krwrxvyxr00007q/T/115811-1568-1em1src/chromedriver_mac32.zip...
info Chromedriver Install Extracting /var/folders/n7/90d6rzz56fdbn_4krwrxvyxr00007q/T/115811-1568-1em1src/chromedriver_mac32.zip to /var/folders/n7/90d6rzz56fdbn_4krwrxvyxr00007q/T/115811-1568-1em1src/chromedriver_mac32
info Chromedriver Install Creating /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-chromedriver/chromedriver/mac...
info Chromedriver Install Copying unzipped binary, reading from /var/folders/n7/90d6rzz56fdbn_4krwrxvyxr00007q/T/115811-1568-1em1src/chromedriver_mac32/chromedriver...
info Chromedriver Install Writing to /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-chromedriver/chromedriver/mac/chromedriver...
info Chromedriver Install /usr/local/lib/node_modules/appium/node_modules/appium-android-driver/node_modules/appium-chromedriver/chromedriver/mac/chromedriver successfully put in place
npm WARN engine xmlbuilder@2.2.1: wanted: {"node":"0.8.x || 0.10.x"} (current: {"node":"0.12.0","npm":"2.5.1"})
npm ERR! Darwin 14.5.0
npm ERR! argv "node" "/usr/local/bin/npm" "install" "-g" "appium@beta"
npm ERR! node v0.12.0
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE

npm ERR! ios-test-app@2.3.0 install: `./build-js/install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the ios-test-app@2.3.0 install script './build-js/install.js'.
npm ERR! This is most likely a problem with the ios-test-app package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ./build-js/install.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls ios-test-app
npm ERR! There is likely additional logging output above.

**Comment by jlipps** [comment]
@Floodzie2 I deleted your comment as I want to keep this issue a more general one, focused on dev work for 1.5, that's easy to scroll through. can you recreate your NPM issue and add the 1.5 label we created for that purpose. please?

FWIW I also ran into that issue, it's something we are working on

**Comment by imurchie** [comment]
Taking on `appium-logger`.

**Comment by Jonahss** [comment]
taking on `authorize-ios`

**Comment by Floodzie2** [comment]
@jlipps - I've created a new issue but I can't add the 1.5 beta tag to it, I think I need permissions?

https://github.com/appium/appium/issues/5606

**Comment by Jonahss** [comment]
taking on `appium-xcode`, relinquishing `authorize-ios` back into the work pool

**Comment by imurchie** [comment]
Taking on `node-simctl`.

**Comment by Jonahss** [comment]
taking `authorize-ios` again

**Comment by Jonahss** [comment]
taking `appium-remote-debugger`

**Comment by imurchie** [comment]
taking on `node-mobile-json-wire-protocol`.

**Comment by Jonahss** [comment]
taking `appium-doctor`

**Comment by Jonahss** [comment]
putting `appium-doctor` back into the pool and taking `appium-ios-simulator` since I have to touch that project right now anyways.

**Comment by imurchie** [comment]
taking on `appium-instruments`

**Comment by moizjv** [comment]
taking on `appium-android-driver`

**Comment by imurchie** [comment]
On to `appium-base-driver`

**Comment by moizjv** [comment]
I will let @scottdixon continue with android-driver, in mean time I will finish adding CI to appium-android-bootstrap and uiautomator

**Comment by imurchie** [comment]
Taking on `appium-express`

**Comment by imurchie** [comment]
Taking `appium-uiauto`

**Comment by imurchie** [comment]
Taking on `appium-ios-logs`

**Comment by scottdixon** [comment]
Taking appium-doctor

**Comment by imurchie** [comment]
On to `jsonwp-proxy`

**Comment by scottdixon** [comment]
Taking `appium-fake-driver`

**Comment by imurchie** [comment]
Taking over `appium-ios-simulator` from @Jonahss

**Comment by saikrishna321** [comment]
@imurchie  will  appium 1.5 v help us running ios tests in parallel ?

**Comment by imurchie** [comment]
@saikrishna321 No, the limitation that Apple has, that only allow a single automation task to occur at one time, remains, unfortunately.

**Comment by scottdixon** [comment]
Returning `appium-fake-driver` - low priority.
Taking `appium-chromedriver`

**Comment by anshu1987** [comment]
@imurchie : I want to use appium source code  into the existing appium setup.
For eg: I have installed Appium 1.3 into my system which is up and running, now Appium 1.4 source code is available without the exe.
How can i use it? Any related article you want to point out.

**Comment by saikrishna321** [comment]
@imurchie  can you check this comment, @aluedeke was able to run the tests in parallel ios devices in the same mac machine with latest xcode 

https://github.com/appium/appium/issues/273#issuecomment-108855158

**Comment by imurchie** [comment]
@anshu1987 You need to install the version of the Appium server from source, and then point to it from inside the Appium application, through the "Developer Settings" panel (there should be a "Use External Appium Package" field that you can fill in with the link to the server).

**Comment by imurchie** [comment]
@saikrishna321 Ah. Yes, it should help. The server can handle multiple sessions at once (which 1.4 and below cannot do). If you can get Instruments to properly work with multiple sessions (some people can, some people cannot) then it should work.

**Comment by imurchie** [comment]
Moving on to `appium-uiautomator`

**Comment by scottdixon** [comment]
Taking `appium-ios-driver`

**Comment by imurchie** [comment]
Working on `appium-android-bootstrap`

**Comment by imurchie** [comment]
Taking on `appium-selendroid-driver`.
